services/provider/server/server.go

```diff
@@ -553,13 +557,18 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 	var storageClassName string
 	storageClassData := map[string]string{}
 
+	storageClassRequestHash := strings.TrimPrefix(storageClassRequest.Name, "storageclassrequest-")
+	nodeSecretName := scrCephClientCsiSecretName("node", storageClassRequestHash)
+	provisionerSecretName := scrCephClientCsiSecretName("provisioner", storageClassRequestHash)
```

Comment:
Why are we generating names for client types that we are not sure would even have secrets provided for? 
The source of truth should be the resource list, not the code.

Reply:
..... we are sure, though. Both StorageClasses are using CSI provisioners, which means they will have the requisite Secrets. I'll also add that we are generating the resource list to send to the consumer from this point, so it's not like we're just passing a 1-to-1 list of extant resources.